ckanext/canada/templates/error_document_template.html

```diff
@@ -2,33 +2,31 @@
 
 {% block page_heading %}
   {% block subtitle %}
-    {% if g.debug %}
-      {{ gettext('Error %(error_code)s', error_code=code) }}
-    {% else %}
+    {% if g.is_registry %}
       {{ _("Open Government Registry") }}
+    {% else %}
+      {% if code == 404 or code == 403 %}
+        {{ _('We couldn\'t find that Web page') }}
+      {% else %}
+        {{ _('We encountered an error') }}
+      {% endif %}
```

Comment:
we don't show error text on the registry?

Reply:
@wardi yeah our error page is bad. It is better now in the CDTS stuff. Idk why we output a custom misc message for 404/403. Was probably a decision long ago.